app/assets/stylesheets/custom.css.scss

```diff
@@ -365,3 +365,24 @@ table td {
   overflow: auto;
   margin-left: 28%;
 }
+
+.payment {
+  font-family: monospace;
+  width: 45%;
```

Comment:
Properties should be ordered font-family, height, margin-left, overflow, padding, width